plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanAParserBaseVisitor.java

```diff
@@ -0,0 +1,357 @@
+// ANTLR GENERATED CODE: DO NOT EDIT
+package org.elasticsearch.plan.a;
```

Comment:
> Then maybe hipchat or an email list is a more suitable place.

It seems you imply those can be ignored, which is weird to me. 

>  In this broken dev culture

This is exactly what I mean by the need for being polite. I'm signing off as I think you understood what I had to say.